### PR TITLE
Do NOT throttle app shell

### PR DESCRIPTION
**Pull request by dijs**
This is a work in progress

### CODE REVIEW

File: packages/react-storefront/service-worker/bootstrap.js [hunk 1]
```diff
@@ -73,6 +75,7 @@ function precacheLinks(response) {
  * the webpack client build
  */
 function isPrefetchRampedUp() {
+  return false
```
Comment (ierceg) [inline packages/react-storefront/service-worker/bootstrap.js]: Debug leftover?

File: packages/react-storefront/service-worker/bootstrap.js [hunk 2]
```diff
@@ -98,7 +100,7 @@ function cachePath({ path, apiVersion } = {}, cacheLinks) {
   return caches.open(cacheName).then(cache => {
     cache.match(path).then(match => {
       if (!match) {
-        if (!isPrefetchRampedUp()) {
+        if (!isPrefetchRampedUp() && path !== appShellPath) {
```
Comment (ierceg) [inline packages/react-storefront/service-worker/bootstrap.js]: Shouldn't this be `|| path === appShellPath`? Actually I think that this condition should be moved to allow-prefetch-throttling `if` - so prefetch app shell but don't throttle it.

Comment (dijs) [reply packages/react-storefront/service-worker/bootstrap.js]: Like this instead:
```js
if ('{{allowPrefetchThrottling}}' === 'true' || path === appShellPath) {
          headers['X-Moov-If-Match'] = 'cache-hit'
        }
```

**Comment by markbrocato** [comment]
@dijs @ierceg Look at my latest commit.  I think this is correct.

**Comment by dijs** [comment]
@markbrocato Thanks for cleaning this up, the code was a bit confusing.

**Pull request merged**